packages/rfw/lib/src/flutter/core_widgets.dart

```diff
@@ -269,6 +269,15 @@ Map<String, LocalWidgetBuilder> get _coreWidgetsDefinitions => <String, LocalWid
     );
   },
 
+  'ClipRRect': (BuildContext context, DataSource source) {
+    // not implemented: clipper
```

Comment:
the convention in the rest of the file seems to be to just have the unimplemented parameters be commented out